ansible/roles/riakcs/README.md

```diff
@@ -0,0 +1,39 @@
+# System Overview
+
+For a system overview look [here](https://docs.google.com/document/d/1F5KjtyXmvGcl4nHf6U8_teMgM4ezo0OJPCUN0AOiGo8/edit#heading=h.9iy7nsyi5rzy)
+
+# Configuring RiakCS
+
+1. Run `deploy_proxy` to update Nginx with the new configs to allow RiakCS to communicate with Stanchion
```

Comment:
we still need to proxy the request from the webworkers to the RiakCS cluster, so it still needs to be run. but yeah the documentation should be updated to reflect that